test/benchmark/inference_engine.cc

```diff
@@ -0,0 +1,146 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include <chrono>
+#include <iostream>
+#include <sstream>
+#include <vector>
+
+#include "test/benchmark/inference_engine.h"
+
+#include "tensorflow/core/platform/macros.h"
+#include "tensorflow/core/protobuf/rewriter_config.pb.h"
+#include "tensorflow/core/public/session.h"
+
+#include "ngraph/event_tracing.hpp"
+#include "ngraph_bridge/ngraph_backend_manager.h"
+#include "ngraph_bridge/version.h"
```

Comment:
The order for include s is:
standard files
tf
ngraph
ng-bridge

So #include "test/benchmark/inference_engine.h" should be in the end